python/pyspark/sql/functions.py

```diff
@@ -1081,7 +1081,9 @@ def approx_count_distinct(col: "ColumnOrName", rsd: Optional[float] = None) -> C
     >>> df.agg(approx_count_distinct(df.age).alias('distinct_ages')).collect()
     [Row(distinct_ages=2)]
     """
-    sc = SparkContext._active_spark_context  # type: ignore[attr-defined]
+    sc = SparkContext._active_spark_context
+    assert sc is not None
+    assert sc._jvm is not None
```

Comment:
Nit: (I don't have strong feelings about it)  Doing this in a single line should work just fine and be preferable in the future:

```python
    assert sc is not None and sc._jvm is not None
```